localization/languages/zh-CN.json

```diff
@@ -122,33 +122,33 @@
         "settingsPreferencesHeading": "设置",
         "settingsRestartRequired": "您需要重启以应用这些修改",
         "settingsPrivacyHeading": "隐私设置",
-        "settingsContentBlockingLevel0": null, //missing translation
-        "settingsContentBlockingLevel1": null, //missing translation
-        "settingsContentBlockingLevel2": null, //missing translation
-        "settingsContentBlockingExceptions": null, //missing translation
+        "settingsContentBlockingLevel0": "允许所有广告和追踪者",
+        "settingsContentBlockingLevel1": "禁止第三方广告和追踪者",
+        "settingsContentBlockingLevel2": "禁止所有广告和追踪者",
+        "settingsContentBlockingExceptions": "下列站点的广告将被允许",
         "settingsBlockScriptsToggle": "阻止脚本运行",
         "settingsBlockImagesToggle": "阻止加载图片",
         "settingsEasyListCredit": {
-            "unsafeHTML": null //missing translation
+            "unsafeHTML": "如果继续发生该错误，请访问<a href=\"https://github.com/minbrowser/min\" target=\"_blank\">页面</a>报告错误."
```

Comment:
I think this isn't the right string, settingsEasyListCredit should be this:

```
            "unsafeHTML": "Based on EasyList and EasyPrivacy, created by <a target=\"_blank\" href=\"https://easylist.to/\">The EasyList authors</a> (<a target=\"_blank\" href=\"https://creativecommons.org/licenses/by-sa/3.0/legalcode\">view license</a>)."
```

Reply:
Sorry. It's the translation for another "unsafeHTML" tag. Fix it later.